src/clr-angular/data/datagrid/datagrid-footer.ts

```diff
@@ -15,9 +15,10 @@ import { Selection, SelectionType } from './providers/selection';
   template: `
         <ng-container
             *ngIf="(selection.selectionType === SELECTION_TYPE.Multi) && (selection.current.length > 0)">
-            <clr-checkbox [clrDisabled]="true" [clrChecked]="true" class="datagrid-foot-select">
-                {{selection.current.length}}
-            </clr-checkbox>
+            <clr-checkbox-container class="datagrid-foot-select disabled">
```

Comment:
Seems like it might be better to use the CSS only version of this since its just visual?

Reply:
I started doing so, yes, but then I realized I would need to generate a unique ID per datagrid for the footer checkbox, and it felt like a pain.

Unless we consider _not_ linking the checkbox and the label is ok because the checkbox is disabled, but I'm pretty sure a11y tools will complain.